control/control.go

```diff
@@ -76,7 +76,8 @@ func (s *SourceParagraph) Maintainers() []string {
 // information that will be eventually put lovingly into the .deb file
 // after it's built on a given Arch.
 type BinaryParagraph struct {
-	Paragraph
+	rfc2822.Paragraph
+
 	Architectures []dependency.Arch `control:"Architecture"`
```

Comment:
Interesting thought -- if we're worried about API breakage, then `pault.ag/go/debian/control` needs to keep shims for all these public methods.  I _think_ we might be able to get clever and have the shim methods use `control:""` and the new methods use `rfc2822:""`.  Worth pondering IMO to see if we can come up with something for that.